core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -362,7 +379,10 @@ public Response getBucket(@HeaderParam("Authorization") final String authorizati
           try {
             List<URIStatus> children = userFs.listStatus(new AlluxioURI(
                 S3RestUtils.MULTIPART_UPLOADS_METADATA_DIR));
-            return ListMultipartUploadsResult.buildFromStatuses(bucket, children);
+            final List<URIStatus> uploadIds = children.stream()
+                .filter((uri) -> uri.getOwner().equals(user))
+                .collect(Collectors.toList());
+            return ListMultipartUploadsResult.buildFromStatuses(bucket, uploadIds);
```

Comment:
if user == null ? ( could it be from no auth configured situation? ) then does that mean the result is empty?